source/code-generation/query-converter/query-converter-test-bench/compare-converted-queries.txt

```diff
@@ -1,4 +1,4 @@
-.. _rm-compare-converted-queries
+.. _rm-compare-converted-queries:
```

Comment:
I'm not sure this is the best name for the topic. The comparison process is entirely manual. Maybe a topic "Test SQL queries" and another one "Test converted queries"? While the processes are similar there are a few nuances including how to specify parameters and view the output.

Reply:
I have a draft of this page still in progress, I think its best to keep 1 single task here, which covers both the SQL query and converted query. There is not much value to testing a single query by itself, the true value of the feature is comparing both the source and destination query and seeing the results in a single place.

Reply:
I renamed the drawer to `Test Converted Queries` and the task page to `Compare Query Results`